### PR TITLE
[hmac] Signoff Checklist (D3/V2 -> D3/V3)

### PR DESCRIPTION
**Pull request by cindychip**
DV is aligned with new RTL optimization. Coverage is approved.
Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/ip/hmac/data/hmac.prj.hjson [hunk 1]
```diff
@@ -7,6 +7,7 @@
     version:            "0.5",
     life_stage:         "L1",
```
Comment (sjgitty) [inline hw/ip/hmac/data/hmac.prj.hjson]: Do you want to fully sign this off? If so, this should move to L2 for Sign Off

File: hw/ip/hmac/doc/checklist.md [hunk 2]
```diff
@@ -199,18 +199,18 @@ Documentation | [DESIGN_DELTAS_CAPTURED_V3][]     | N/A         |
...
 Regression    | [NIGHTLY_REGRESSION_AT_100][]     | Done        |
-Coverage      | [CODE_COVERAGE_AT_100][]          | In Progress | [#820][] pending. prim_fifo_sync revised [common_cov_excl.el][] HMAC excl to be added
+Coverage      | [CODE_COVERAGE_AT_100][]          | Done        | [pr #945][]. prim_fifo_sync revised [hmac_cov_excl.el][]
 Coverage      | [FUNCTIONAL_COVERAGE_AT_100][]    | Done        |
 Issues        | [NO_ISSUES_PENDING][]             | Done        |
 Code Quality  | [NO_TOOL_WARNINGS_THROWN][]       | Done        | Xcelium warnings on C code
 Integration   | [PRE_VERIFIED_SUB_MODULES_V3][]   | N/A         |
-Review        | Reviewer(s)                       | Not Started | @eunchan @sjgitty @sriyerg
-Review        | Signoff date                      | Not Started |
+Review        | Reviewer(s)                       | Done        | @eunchan @sjgitty @sriyerg
```
Comment (sjgitty) [inline hw/ip/hmac/doc/checklist.md]: If you want an official sign off on this, please convene a physical review of the
items; those names should be the ones as reviewers in that case. If you want
to finish off before Wednesday, you can use Tim as the other design reviewer.

Comment (cindychip) [reply hw/ip/hmac/doc/checklist.md]: Thanks for the reminder and suggestion Scott. I will send out an invite for HMAC signoff review.

File: hw/ip/hmac/data/hmac.prj.hjson (hunk 1 above)
Comment (cindychip) [reply hw/ip/hmac/data/hmac.prj.hjson]: Yes I would like to fully sign this off, will update the change, thanks for the reminder

File: hw/ip/hmac/data/hmac.prj.hjson [hunk 3]
```diff
@@ -5,8 +5,8 @@
 {
     name:               "hmac",
     version:            "0.5",
-    life_stage:         "L1",
+    life_stage:         "L2",
     design_stage:       "D3",
-    verification_stage: "V2",
+    verification_stage: "V3",
     notes:              "D3 commit id: 3471a47c96ba87c77de7b14c39f4faf71a839283",
```
Comment (cindychip) [inline hw/ip/hmac/data/hmac.prj.hjson]: I will add another PR to update the V3 commit ID once this PR is merged.

Comment (sjgitty) [reply hw/ip/hmac/data/hmac.prj.hjson]: make sure you use the commit_id field.

Comment (cindychip) [reply hw/ip/hmac/data/hmac.prj.hjson]: Will do, thank you for the reminder.

**Pull request merged**